esp/esdllib/esdl_def_helper.cpp

```diff
@@ -113,6 +113,58 @@ void EsdlDefinitionHelper::setTransformParams( EsdlXslTypeId xslId, IProperties
     parameters.setValue( xslId, params );
 }
 
+void removeEclHiddenStructs(IPropertyTree &depTree)
+{
+    Owned<IPropertyTreeIterator> it = depTree.getElements("*[@ecl_hide='1']");
+    ForEach(*it)
+        depTree.removeTree(&it->query());
+}
+void removeEclHiddenElements(IPropertyTree &depTree)
+{
+    Owned<IPropertyTreeIterator> it = depTree.getElements("*");
+    ForEach(*it)
```

Comment:
I'm sure this process is well though out but it's hard for me to visualize, since there's two loops within a parent loop. Let's just make sure this is the most efficient way to remove all hidden structs.